requirements.txt

```diff
@@ -1,23 +1,23 @@
 coala==0.4.2.dev20160316074840
-setuptools>=19.2
-munkres3>=1.0.5.5
-pylint>=1.5.2
-language-check>=0.7.5
-autopep8>=1.2.1
-eradicate>=0.1.6
-autoflake>=0.6.6
-restructuredtext_lint>=0.14.0
-proselint>=0.3.5
-cpplint>=0.0.7
-isort>=4.2.2
-html-linter>=0.3.0
+setuptools~=19.2
+munkres3~=1.0.5.5
+pylint~=1.5.2
+language-check~=0.7.5
+autopep8~=1.2.1
+eradicate~=0.1.6
+autoflake~=0.6.6
+restructuredtext_lint~=0.14.0
+proselint~=0.3.5
+cpplint~=0.0.7
+isort~=4.2.2
+html-linter~=0.3.0
 # Do *not* use 0.3 which is not backwards compatible to common clang versions
 libclang-py3==0.2
-guess-language-spirit>=0.5.2
+guess-language-spirit~=0.5.2
 radon==1.2.2
```

Comment:
Then we need a comment here saying that

On Thu, Mar 17, 2016 at 9:06 PM, Lasse Schuirmann <notifications@github.com>
wrote:

> In requirements.txt
> <https://github.com/coala-analyzer/coala-bears/pull/250#discussion_r56524448>
> :
>
> >  # Do *not* use 0.3 which is not backwards compatible to common clang versions
> >  libclang-py3==0.2
> > -guess-language-spirit>=0.5.2
> > +guess-language-spirit~=0.5.2
> >  radon==1.2.2
>
> because radon uses a == that broke our requirements IIRC
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/250/files/c98aad5a1704c6159b4fcb817413aef7b0973a38#r56524448>
>